### DISCUSSION TITLE
How to change the color of the slide bar

### DISCUSSION THREAD

**Discussion by hi123098** [jspsych/jsPsych#2576] [Q&A] [answered]
Hi, 
I am quite new to coding. I am using the plugin "html-slider-response" and I would like to change the color of the slider bar, but I am having some trouble doing so... May I ask what functions should I use or how can I do so? 
Thanks a lot!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @hi123098,

You can control the appearance of the slider by setting the CSS of the `<input type="range">` elements.

[This is a neat tool](https://www.cssportal.com/style-input-range/) that lets you select options and then generates the corresponding CSS.

You should be able to copy/paste the CSS into the `<head>` section of the experiment, inside a `<style></style>` tag. You may need to remove the CSS styles related to the range element in the `jspsych.css` file.
